src/Cms/AppTranslations.php

```diff
@@ -119,18 +120,13 @@ public function setCurrentTranslation(string $translationCode = null): void
     /**
      * Set locale settings
      *
-     * @internal
+     * @deprecated 3.5.0 Use \Kirby\Toolkit\Locale::set() instead
+     *
      * @param string|array $locale
      */
     public function setLocale($locale): void
     {
-        if (is_array($locale) === true) {
-            foreach ($locale as $key => $value) {
-                setlocale($key, $value);
-            }
-        } else {
-            setlocale(LC_ALL, $locale);
-        }
+        Locale::set($locale);
```

Comment:
To be honest, I wouldn't deprecate this one. It's an unnecessary breaking change. I'd leave the method and just call `Locale::set()` inside as you do. And then I also would leave the user calls in this class/trait to `$this->setLocale()`. In that way we only have to change something in one place (if we have ever changes again here), instead of three places.

Reply:
A deprecation isn't a breaking-change though. I also wouldn't add a deprecation warning to this, it's just a reminder for us to clean this up at some point.

In my opinion, keeping the method around forever will be confusing, especially given that the `App` class has so many methods already.

Reply:
Well, eventually it is a breaking change - deprecating in the end means we will remove it at some point. But yes, at the end it's probably a matter of taste whether to reduce the number of methods on the `App` class here.

Reply:
I think it's good to keep it as a reminder that we shouldn't use it anymore. Nothing stops us from keeping it around until 4.0